seers/detect.py

```diff
@@ -0,0 +1,74 @@
+from imutils.video import VideoStream
```

Comment:
Don't know if thats necessary within the code because you can look at the code and see where it is being used (although there is one import that i just realized i'm no longer using, will delete). I definitely will mention it in the technical documentation/write up I'm going to do about this .

Reply:
The documentation isn't about the import, it's for the file. Sorry, that's probably not super clear since the review highlights that line particularly.